src/nidigital/metadata/enums.py

```diff
@@ -792,6 +792,22 @@
             }
         ]
     },
+    'VectorMemoryType': {
```

Comment:
There is no reason for these to be in the Python metadata, they aren't for public use.

Reply:
Reverted it and I've got an internal PR to update the internal metadata so that this won't be included next time.
We'll just skip updating this metadata this time around, since there's no other changes.

I considered updating the internal ~template~ expander to exclude private access functions, attributes and enums,, but then I remembered the special case of the "internal" SetRuntimeEnvironment function that we include in our metadata and didn't want to risk breaking a similar case with "private" access.